tests/expose_test.go

```diff
@@ -430,40 +430,30 @@ var _ = Describe("Expose", func() {
 
 				Eventually(getStatus, 60, 1).Should(Equal(k8sv1.PodSucceeded))
 
-				By("Stopping the running VM.")
-				virtctl := tests.NewRepeatableVirtctlCommand("stop", "--namespace", vm.Namespace, vm.Name)
-				err = virtctl()
-				Expect(err).ToNot(HaveOccurred())
-
-				By("Verifying the VMI is not running.")
+				By("Retrieving the current VMI UID, to be compared with the new UID after restart.")
```

Comment:
I don't think this should be a 'By'. 'By' is test step. this is just an internal operation that you need to do.